jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp

```diff
@@ -275,12 +275,6 @@
     <td class="userinfo">
       <div>
         <p>
-          <spring:message var="onlineTip" code="label.tips.user_online"/>
-          <spring:message var="offlineTip" code="label.tips.user_offline"/>
-          <c:set var="online" value='<i class="icon-online" title="${onlineTip}"></i>'/>
-          <c:set var="offline" value='<i class="icon-offline" title="${offlineTip}"></i>'/>
-          <jtalks:ifContains collection="${usersOnline}" object="${post.userCreated}"
```

Comment:
Sorry. Clean all unused from postcontroller and postcontrollertest. But we must keep sessionRegistry in context it's used in other parts